lib/opentelemetry/types.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+const TokenConsumedSchema = z.object({
```

Comment:
this schema is used in type check at each instrumentation because the structured logs will be parsed before analysis